src/vue/modules/requests/create-asset-requests/components/requests-table.vue

```diff
@@ -31,7 +31,7 @@
             :key="request.id"
           >
             <td :title="request.assetCode">
-              {{ request.assetCode }}
+              {{ request.assetCode.id }}
```

Comment:
Fix this bug in the title(one line above)

Reply:
fixed